File: core_backend/core_backend/Models/ClassifiedPeriod.cs
using System.ComponentModel.DataAnnotations;

namespace core_backend.Models
{
    public class ClassifiedPeriod
    {
        [Key]
        public string Uuid { get; set; }
        public string Origin { get; set; }
        public long StartDate { get; set; }
        public long EndDate { get; set; }
        public bool Confirmed { get; set; }
        public long CreatedOn { get; set; }
        public long DeletedOn { get; set; }
        public string UserId { get; set; }
    }
}

File: backend/models/classified_period.go
package models

type ClassifiedPeriod struct {
	Uuid      string `gorm:"primary_key" json:"uuid"`
	Origin    string `json:"origin"`
	StartDate int64  `json:"startDate"`
	EndDate   int64  `json:"endDate"`
	Confirmed bool   `json:"confirmed"`
	CreatedOn int64  `json:"createdOn"`
	DeletedOn int64  `json:"deletedOn"`
	UserId    int64
}
